contrib/slime-repl.el

```diff
@@ -280,13 +280,14 @@ This is set to nil after displaying the buffer.")
       (slime-save-marker slime-output-start
         (slime-propertize-region '(face slime-repl-output-face
                                         slime-repl-output t
-                                        rear-nonsticky (face))
+                                        read-only t
+                                        rear-nonsticky t)
           (let ((inhibit-read-only t))
-	    (insert-before-markers string)
-	    (when (and (= (point) slime-repl-prompt-start-mark)
-		       (not (bolp)))
-	      (insert-before-markers "\n")
-	      (set-marker slime-output-end (1- (point))))))))
+	          (insert-before-markers string)
+	          (when (and (= (point) slime-repl-prompt-start-mark)
+		                   (not (bolp)))
+	            (insert-before-markers "\n")
+	            (set-marker slime-output-end (1- (point))))))))
```

Comment:
Looks like some spaces-to-tabs conversions sneaked in here. You might like to run `untabify` on the buffer to put that back and maybe even make that default with an emacs config line like

```lisp
(add-hook 'lisp-mode-hook (defun my-lisp-mode-hook () (setq indent-tabs-mode nil)))
```